src/core/agent/modules/signify/signifyApi.ts

```diff
@@ -149,9 +150,16 @@ export class SignifyApi {
   }
 
   async getOobi(name: string): Promise<string> {
+    const identifiers = await AriesAgent.agent.identifiers.getIdentifiers();
+    const identifierMetadata = identifiers.find(
+      (identifier) => identifier.signifyName === name
+    );
```

Comment:
We need to fetch from the DB based on tag - rather than fetching every identifier and filtering. We have SQL running so we need to take advantage of it

Reply:
We don't have the `signifyName` tag at the moment. Should we add it?

Reply:
yes please :)